### PR TITLE
[CHORE] flake8 issues E402, E721 and F821

### PR DESCRIPTION
**Pull request by dni**
for more details look into commit messages.

could not fix all E402 issues, the last one needed following refactor.
https://github.com/lnbits/lnbits/pull/1876

E721 only popped up for python 3.9 not 3.10

### CODE REVIEW

File: lnbits/wallets/lnpay.py [hunk 1]
```diff
@@ -139,25 +136,26 @@ async def paid_invoices_stream(self) -> AsyncGenerator[str, None]:
...
-            raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
-
-        lntx_id = data["data"]["wtx"]["lnTx"]["id"]
-        r = await self.client.get(f"/lntx/{lntx_id}?fields=settled")
-        data = r.json()
-        if data["settled"]:
-            await self.queue.put(lntx_id)
-
-        raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
+        # text: str = await request.get_data()
```
Comment (motorina0) [inline lnbits/wallets/lnpay.py:143]: dead code?

File: lnbits/wallets/opennode.py [hunk 2]
```diff
@@ -136,18 +133,20 @@ async def paid_invoices_stream(self) -> AsyncGenerator[str, None]:
...
-        charge_id = data["id"]
-        x = hmac.new(self.auth["Authorization"].encode("ascii"), digestmod="sha256")
-        x.update(charge_id.encode("ascii"))
-        if x.hexdigest() != data["hashed_order"]:
-            logger.error("invalid webhook, not from opennode")
-            raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
-
-        await self.queue.put(charge_id)
-        raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
+        # data = await request.form  # type: ignore
```
Comment (motorina0) [inline lnbits/wallets/opennode.py:140]: - why disable code?
- will it later be re-enabled? if not, then it should be remove.

**Comment by motorina0** [comment]
- why is `isinstance(something, str)` preferred to `type(something) == str` ?

**Comment by jackstar12** [comment]
> - why is `isinstance(something, str)` preferred to `type(something) == str` ? 

If something wasnt an actual `str` but maybe a `claas MyStr(str)` the latter would return False while the `isinstance` check returns True. This can lead to unexpected bugs

File: lnbits/wallets/lnpay.py (hunk 1 above)
Comment (dni) [reply lnbits/wallets/lnpay.py:143]: lets open a issue for those 2, this code does not work currently thats why i commented it out, at some point some need to reimplement it i guess.

File: lnbits/wallets/opennode.py (hunk 2 above)
Comment (dni) [reply lnbits/wallets/opennode.py:140]: same here

**Review by talvasconcelos — APPROVED** [review]
apart from the commented out code vs deleting, it looks fine

**Pull request merged**